app/components/account/action/add/account-create-basic-dialog.component.spec.ts

```diff
@@ -0,0 +1,316 @@
+import { DebugElement, NO_ERRORS_SCHEMA } from "@angular/core";
```

Comment:
Let's not name new component with this `basic-dialog` suffix.
Just have `batch-account-create-component` should be good(include batch too)